samples/msal-node-samples/on-behalf-of-distributed-cache/src/RedisClientWrapper.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ICacheClient } from "@azure/msal-node";
+import { RedisClientType } from "redis";
+
+const CACHE_TTL = 60 * 60 * 24; // 24 hours
```

Comment:
We should ask developers to set this to whatever token lifetime they get from the client app. Ideally MSAL would help with this, but we don't have that implemented :(

Reply:
The access token lifetime (at least today) is varied though, no? This is a general default setting, but I think we can set the TTL to what comes from the token request response (as you mentioned in your comment above), and fallback to this as default just in case